snmp/metadata.csv

```diff
@@ -8,6 +8,15 @@ datadog.snmp_traps.invalid_packet,count,,packet,,The number of invalid SNMP Trap
 datadog.snmp_traps.received,count,,packet,,The total number of SNMP Traps received.,0,snmp,,
 datadog.snmp_traps.traps_not_enriched,count,,packet,,The number of SNMP Traps that were not enriched.,0,snmp,,
 datadog.snmp_traps.vars_not_enriched,count,,,,The number of SNMP Traps variables that were not enriched.,0,snmp,,
+snmp.IAmACounter32,gauge,,,,,0,snmp,,
+snmp.IAmACounter64,gauge,,,,,0,snmp,,
+snmp.IAmAGauge32,gauge,,,,,0,snmp,,
+snmp.IAmAOctetStringFloat,gauge,,,,,0,snmp,,
+snmp.IAmAnInteger,gauge,,,,,0,snmp,,
+snmp.aStringFloatValue,gauge,,,,Custom metric,0,snmp,,
+snmp.aTemperatureValueInferred,gauge,,,,Custom metric,0,snmp,,
```

Comment:
I think we don't need to add `aggregator.assert_metrics_using_metadata()` for some test in `test_profiles.py` that assert fake snmp metric like those ones.

Reply:
And also some other metrics define here: https://github.com/DataDog/integrations-core/blob/896cf24644445392ccbef2af783cdaf57ae54f0e/snmp/tests/common.py#L53-L195

That might not be real metrics we find in actual profiles from https://github.com/DataDog/integrations-core/tree/master/snmp/datadog_checks/snmp/data/default_profiles

Reply:
A possible way to clean-up is to verify that new metrics added in metadata.csv are indeed present in [default_profiles](https://github.com/DataDog/integrations-core/tree/master/snmp/datadog_checks/snmp/data/default_profiles)

Reply:
Thanks for the comments, I reviewed added metrics and deleted the ones that are not present in default protocols